### PR TITLE
EW-838 Connection Common Cartridge Service to Files Storage

### PR DESCRIPTION
**Pull request by psachmann**
# Description
Adding e new deployment role for the Common Cartridge microservice.

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/EW-838
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/5078

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/playbook_rollout.yml [hunk 1]
```diff
@@ -49,3 +49,5 @@
     - role: post_deployment
     - role: moin-schule-sync
       when: WITH_MOIN_SCHULE is defined and WITH_MOIN_SCHULE|bool
+    - role: common-cartridge
+      when: WITH_COMMON_CARTRIDGE is defined and WITH_COMMON_CARTRIDGE|bool
```
Comment (mamutmk5) [inline ansible/playbook_rollout.yml:53]: It's not necessary to check the when condition as is also at add as conditions for the job in the ansible role
```suggestion
    - role: common-cartridge
```
And put it after the `schulcloud-client-core` role and bevore the `post_deployment` role. 
The `post_deployment` ansible role shoud be the last role, it marks that the deployment is finished

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/playbook_rollout.yml (hunk 1 above)
Comment (psachmann) [reply ansible/playbook_rollout.yml]: Done

File: ansible/playbook.yml [hunk 2]
```diff
@@ -45,3 +45,4 @@
     - ingress
     - post_deployment
     - moin-schule-sync
+    - common-cartridge
```
[the remark this answers is not in the document]
Comment (psachmann) [reply ansible/playbook.yml]: Done

**Comment by mamutmk5** [comment]
Importen for Merging this PR you must first merge hpi-schul-cloud/schulcloud-server#5078 so that the ansible role is there then add the ansible role add to the playbocks works with out an error as the playbock is rolled out to dev

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**